src/components/App.js

```diff
@@ -0,0 +1,11 @@
+import ButtonPanel from './ButtonPanel';
+import Display from './Display';
+
+const App = () => (
+  <div className="app">
+    <Display result="999" />
+    <ButtonPanel />
+  </div>
+);
+
```

Comment:
You are required to wrap these 2 components using `React.Fragments`, instead of `div` [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png)]